gravitee-apim-cypress/cypress/integration/rest-api/apis/import-update-tests.ts

```diff
@@ -25,20 +25,13 @@ import {
 } from '../../../commands/management/api-management-commands';
 import { getPage, getPages } from '../../../commands/management/api-pages-management-commands';
 import { ApiImportFakers } from '../../../fixtures/fakers/api-imports';
-import {
-  ApiMetadataFormat,
-  ApiPlanSecurityType,
-  ApiPlanStatus,
-  ApiPlanType,
-  ApiPlanValidationType,
-  ApiPrimaryOwnerType,
-  ApiVisibility,
-} from '@model/apis';
+import { ApiMetadataFormat, ApiPrimaryOwnerType, ApiVisibility } from '@model/apis';
 import { getPlans } from '../../../commands/management/api-plans-management-commands';
 import { GroupFakers } from '../../../fixtures/fakers/groups';
 import { createGroup, deleteGroup, getGroup } from '../../../commands/management/environment-management-commands';
 import { createUser, deleteUser } from '../../../commands/management/user-management-commands';
 import { createRole, deleteRole } from '../../../commands/management/organization-configuration-management-commands';
```

Comment:
You probably want to use TS path aliases here as well (e.g. @commands/...)

Reply:
I missed this one, nice catch. Fixed it. :+1: